client-app/shared/layout/components/footer/footer.vue

```diff
@@ -62,6 +62,7 @@
           <FooterLink to="/rebates" v-t="'shared.layout.footer.rebates_link'"></FooterLink>
           <FooterLink to="/replacement-parts" v-t="'shared.layout.footer.replacement_parts_link'"></FooterLink>
           <FooterLink to="/demo-landing" v-t="'shared.layout.footer.demo_landing_link'"></FooterLink>
+          <FooterLink v-if="isDevelopment" :to="{ name: 'DevUIKit' }" v-t="'shared.layout.footer.ui_kit_link'" />
```

Comment:
Maybe this link shouldn't be localized as it is used only in development. Also it's no need to extend locale file with non-production strings.